frontend-react/cypress/support/component-index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
We don't need this right now because we're not running component tests through Cypress, but I feel like that _could_ come in handy eventually -- especially for style updates and accessibility.  (Jest is fine for this sort of thing, but that just tests the resultant markup whereas this'd actually render components into a browser.)